src/boards/boards.c

```diff
@@ -93,6 +95,8 @@ void board_init(void)
 #if ENABLE_DCDC_1 == 1
   NRF_POWER->DCDCEN = 1;
 #endif
+  // Make sure any custom inits are performed
+  extern_board_init();
```

Comment:
need to change as well.

Reply:
Sorry, to early in the morning for this stuff =)

Reply:
no problems at all, me as well without coffee